Annotato/Annotato/ViewModels/Document/DocumentAnnotationViewModel.swift

```diff
@@ -29,4 +46,31 @@ class DocumentAnnotationViewModel {
     var startingAnnotationType: AnnotationType? {
         isNew ? self.parts[0].annotationType : nil
     }
+
+    func updateLocation(documentViewPoint: CGPoint, parentView: UIView?) {
+        guard let parentView = parentView else {
+            return
+        }
+        guard let pdfView = parentView.superview?.superview
+        as? DocumentPdfView else {
+            return
+        }
+
+        // Get coordinates in terms of the visible view page space
+        let visibleViewCoordinates = parentView.convert(
+            documentViewPoint, to: parentView.superview?.superview)
+
+        // Get the current page that this annotation is at
+        guard let currPage = pdfView.page(
+            for: visibleViewCoordinates, nearest: true) else {
+            return
+        }
+        // Get the page space coordinates
+        let pageSpaceCoordinates = pdfView.convert(visibleViewCoordinates, to: currPage)
+
+        // Assign all the new values to update
+        self.associatedPage = currPage
+        self.coordinatesInPageSpace = pageSpaceCoordinates
+        self.coordinatesInDocumentSpace = documentViewPoint
```

Comment:
Is there a reason why some things are named `coordinates` and others are named `point`, even though they seem to represent the same idea?